modules/sharedaddy/sharing-sources.php

```diff
@@ -724,7 +724,18 @@ public function display_footer() {
 			if ( ! $locale ) {
 				$locale = 'en_US';
 			}
-			?><div id="fb-root"></div><script>(function(d, s, id) { var js, fjs = d.getElementsByTagName(s)[0]; if (d.getElementById(id)) return; js = d.createElement(s); js.id = id; js.src = '//connect.facebook.net/<?php echo $locale; ?>/sdk.js#xfbml=1&appId=249643311490&version=v2.3'; fjs.parentNode.insertBefore(js, fjs); }(document, 'script', 'facebook-jssdk'));</script><?php
+			/**
+			 * Filter the App ID used in the official Facebook Share button.
+			 *
+			 * @since 3.8.0
+			 *
+			 * @param int $fb_app_id Facebook App ID. Default to empty.
+			 */
+			$fb_app_id = absint( apply_filters( 'jetpack_sharing_facebook_app_id', '' ) );
```

Comment:
Let's continue to default to the WP app ID, since this has to be synced back there.